### ISSUE TITLE
VM: Reintegrate Slow Tests / Other

### ISSUE THREAD

**Issue by holgerd77** [ethereumjs/ethereumjs-monorepo#2929]
Will make this a replacement issue for #1576 since the issue still exists and is important and so to give this some better visibility again.

So task is to look into the skipped tests from [vm/test/tester.config.ts](https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/vm/test/tester/config.ts) and slowly re-activate deactviated tests.

A special case here - also a reason for a new issue, to point this out more prominently - is the reactivation of the performance tests. We have done a lot of performance work, and we should see what tests are now running fastly enough that we can re-integrate. And on the other hand, *not* yet sufficiently running tests are a fantastic ground to indentify still remaining issues, and these might then also deserve a dedicated *separate* issue to dig deeper into the problem.

**Comment by jochem-brouwer** [comment]
Yes, regarding performance, will just post this here since I also just posted it on our discord:

I just got an interesting idea here: we can use the entire test suite to benchmark our EVM. The idea is: we look at the cumulative gas used in the entire test, divide this by time, and then sort these tests based upon that. By EVM design this should be constant (but obviously is not). If we have huge outliers then we know where to look on how to optimize our EVM 😄
